paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1305,6 +1321,76 @@ def diagnose_why_instance_is_stuck(
             print(f"    {line}")
     print("")
 
+    if should_ping_for_unhealthy_pods:
+        maybe_ping_for_unhealthy_pods(
+            service, instance, cluster, git_sha, status, notify_fn
+        )
+
+
+already_pinged = False
```

Comment:
nit: I'm not sure I like this global state even though it's technically okay since this is a command. Ideally, the runner is a class with `should_ping` as instance state.

Reply:
Yeah this definitely feels a little bad to me, but I didn't feel like refactoring these functions into methods. I'll see how difficult this refactor would be.

Reply:
I guess I could pull this into its own OnlyPingOnce class or something, but refactoring this so that the `already_pinged` variable lives on the MarkForDeploymentProcess instance seems not worth it. I think I'm okay with this small bit of global state.